### PR TITLE
Cleans remaining env vars in server config

### PR DESCRIPTION
**Pull request by Rotorsoft**
See ticket

## Link to Issue
Closes: #7897

## Description of Changes
- Moves remaining vars to config

### CODE REVIEW

File: packages/commonwealth/server/controllers/server_groups_methods/refresh_community_memberships.ts [hunk 1]
```diff
@@ -166,9 +164,9 @@ async function computeMembership(
   balances: OptionsWithBalances[],
 ): Promise<ComputedMembership> {
   const { requirements } = currentGroup;
-  const { isValid, messages } = await validateGroupMembership(
+  const { isValid, messages } = validateGroupMembership(
     address.address,
-    requirements,
+    requirements as Requirement[],
```
Comment (timolegros) [inline packages/commonwealth/server/controllers/server_groups_methods/refresh_community_memberships.ts:169]: How was this type error even making it through builds/CI?

**Review by timolegros — APPROVED** [review]
LGTM. Let's follow-up with this soon: https://github.com/hicommonwealth/commonwealth/issues/7914

File: packages/commonwealth/server/controllers/server_groups_methods/refresh_community_memberships.ts (hunk 1 above)
Comment (Rotorsoft) [reply packages/commonwealth/server/controllers/server_groups_methods/refresh_community_memberships.ts:169]: probably strict mode linting? not sure

**Comment by Rotorsoft** [comment]
> LGTM. Let's follow-up with this soon: #7914

sounds good... we should also discuss some naming conventions for env vars, and group feature flags at the application level using a grouping like FEATURES: {}

File: packages/commonwealth/server/controllers/server_groups_methods/refresh_community_memberships.ts (hunk 1 above)
Comment (timolegros) [reply packages/commonwealth/server/controllers/server_groups_methods/refresh_community_memberships.ts:169]: Why would strict mode linting stop a type error from being caught? Shouldn't we investigate further? Might point to an issue with our type-checking which we should fix immediately if it exists.

**Review by rbennettcw — APPROVED** [review]

File: packages/commonwealth/server/controllers/server_groups_methods/refresh_community_memberships.ts (hunk 1 above)
Comment (Rotorsoft) [reply packages/commonwealth/server/controllers/server_groups_methods/refresh_community_memberships.ts:169]: Resolving to merge... we can continue this topic in the retro

**Pull request merged**